numpy/core/src/multiarray/compiled_base.c

```diff
@@ -116,10 +116,10 @@ arr_bincount(PyObject *NPY_UNUSED(self), PyObject *args, PyObject *kwds)
     }
     else {
         minlength = PyArray_PyIntAsIntp(mlength);
```

Comment:
It is very strong.  Note that I did change the docstring to pretend this is the case...